lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -214,57 +188,27 @@
         return platform.tagPublicLibraryCode(f, name);
     };
 
-    const __resolveLocaleBestFit = new IntlCache();
-    const resolveLocaleBestFit = function (locale) {
-        var resolvedLocale = __resolveLocaleBestFit.get(locale);
-        if (resolvedLocale === undefined) {
-            resolvedLocale = platform.resolveLocaleBestFit(locale);
-            // REVIEW(jahorto): when ICU returns null here, fall back to the JS version of resolveLocaleLookup
-            // rather than the native version to take advantage of the JS implementation of BestAvailableLocale
-            if (resolvedLocale === null) {
-                resolvedLocale = resolveLocaleLookup(locale);
-            }
-            // If resolvedLocale is undefined, cache that we got undefined
-            // so we don't try to resolve for `locale` in future.
-            __resolveLocaleBestFit.set(locale, resolvedLocale === undefined ? LANGUAGE_NOT_FOUND : resolvedLocale);
-        } else if (resolvedLocale === LANGUAGE_NOT_FOUND) {
-            resolvedLocale = undefined;
-        }
-
-        // our best fit matcher for WinGlob will return the default locale if given an unsupported locale
-        if (isPlatformUsingWinGlob) {
-            const defaultLocale = GetDefaultLocale();
-            if (defaultLocale === locale) {
-                return resolvedLocale;
-            } else if (defaultLocale === resolvedLocale) {
-                return undefined;
-            } else {
-                return resolvedLocale;
-            }
-        }
-
-        return resolvedLocale;
-    }
-
     /**
      * Determines the best possible locale available in the system
      * 
      * ECMA-402: #sec-bestavailablelocale
      * 
-     * Result is _not_ cached because it is expected that the caller will cache its result instead
```

Comment:
Does this comment no longer apply? It looks to me like there is no caching of results, unless that is expected to be handled by `availableLocales`?

Reply:
The caching story will need to be re-evaluated in a future pull -- as it stands right now, very little is cached.